addOns/soap/src/main/resources/org/zaproxy/zap/extension/soap/resources/Messages.properties

```diff
@@ -16,18 +16,18 @@ soap.soapactionspoofing.desc=An unintended SOAP operation was executed by the se
 soap.soapactionspoofing.other=An unintended SOAP operation was executed by the server.
 soap.soapactionspoofing.soln=If not required, the SOAPAction attribute should be disabled. If needed, the operation within the SOAPAction and the SOAP body should always be compared before executing any operation. Any mismatch should be regarded as an attack.
 soap.soapactionspoofing.refs=http://www.ws-attacks.org/index.php/SOAPAction_Spoofing
-soap.soapactionspoofing.invalidFormatMsg=Response has an invalid format.
-soap.soapactionspoofing.faultCodeMsg=Server returned a fault code.
-soap.soapactionspoofing.emptyResponseMsg=Response is empty.
-soap.soapactionspoofing.soapactionIgnoredMsg=The SOAPAction header has been ignored.
-soap.soapactionspoofing.soapactionExecutedMsg=The SOAPAction operation has been executed.
+soap.soapactionspoofing.invalid_format=Response has an invalid format.
+soap.soapactionspoofing.fault_code=Server returned a fault code.
+soap.soapactionspoofing.empty_response=Response is empty.
+soap.soapactionspoofing.soapaction_ignored=The SOAPAction header has been ignored.
+soap.soapactionspoofing.soapaction_executed=The SOAPAction operation has been executed.
 
```

Comment:
The key names are probably better with additional periods.

Reply:
I agree, but the way they've been used here: https://github.com/zaproxy/zap-extensions/pull/2637/files#diff-261a723b534a9dd3102b613eb0e62154cbe548e73e1228f5a6c10481efea6622R271 makes underscores more convenient. (The corresponding enum field names are converted to lowercase and appended to MESSAGE_PREFIX.)

Reply:
Okay!